src/controllers/apiAuth.controller.ts

```diff
@@ -0,0 +1,62 @@
+import passport from "passport";
+import passportCustom from "passport-custom";
+import bcrypt from "bcrypt";
+const CustomStrategy = passportCustom.Strategy;
+import { pool } from "../config/database";
+
+async function hashPassword(password: string) {
+  const saltRounds = 10;
```

Comment:
Please set the salt round in an enviromental variable and use it here for safety reasons. You don't want malicious users knowing how many salt rounds you used to hash your password.